task-management/src/main/resources/locale/portlet/taskManagement_en.properties

```diff
@@ -160,8 +162,8 @@ popup.msg.projectNotExist=This project does not exist or has been deleted.
 popup.msg.deleteProject=You are about to delete project <strong>{}</strong> and all its tasks. Please confirm.
 popup.msg.deleteLabel=Are you sure you want to delete <strong>{0}</strong> label?
 popup.msg.cloneProject=You are about to clone project <strong>{}</strong>. Please confirm.
-popup.msg.deleteTask=Are you sure you want to delete this task : '{}' ?
-popup.msg.deleteStatus=Are you sure you want to delete this status
+popup.msg.deleteTask=Are you sure you want to delete this task
```

Comment:
```suggestion
popup.msg.deleteTask=Are you sure you want to delete this task ?
```

Reply:
IMO not needed check https://github.com/exoplatform/task/pull/139/files#diff-a95097cc7a29ad4e66cb1cf86f29748d683d15ac2dc8ae0fda4615b2db93f6b2R526

Reply:
this os a wrong way of going it  question marks are at different locations in some languages such as Spanish. It should be in the resource bundle, not  in code!
@mzorai please make sure this is redone properly